API.md

```diff
@@ -89,13 +121,13 @@ Json document with a quote for amount of tokens to transfer to the provider acco
 
 Example:
 ```
-POST /api/services/initialize
+GET /api/services/initialize
 payload:
 {
     "documentId":"0x1111",
     "serviceId": 0,
-    "datatoken": "",
     "consumerAddress":"0x990922334",
+}
 ```
```

Comment:
I think we should add a 2nd example of an `initialize` call for a "compute" service, which includes additional arguments "computeEnv" and "validUntil"

Reply:
it's not computeEnv anymore, it's been renamed "environment" in a parallel branch/PR, since it's easier to send towards c2d like a passthrough.